nntrainer/models/model_loader.cpp

```diff
@@ -62,8 +62,8 @@ int ModelLoader::loadModelConfigIni(dictionary *ini, NeuralNetwork &model) {
   model.loss_type = (LossType)parseType(
     iniparser_getstring(ini, "Model:Loss", unknown), TOKEN_LOSS);
   const std::string &save_path =
-    iniparser_getstring(ini, "Model:Save_path", "./model.bin");
-  model.setSavePath(model.app_context.getWorkingPath(save_path));
+    iniparser_getstring(ini, "Model:Save_path", "tmp_model.bin");
+  model.setSavePath(save_path);
```

Comment:
I wonder why the prefix of `getWorkingPath` has been removed?

Reply:
`model.setSavePath` internally calls `save_path` so this is not needed here.

Reply:
So, if model has a `getSavePath` later, it will also have to add `getWorkingPath`?

Reply:
`setSavePath` adds it so it won't need `getWorkingPath`